extensions/ql-vscode/src/language-context-store.ts

```diff
@@ -43,7 +43,18 @@ export class LanguageContextStore extends DisposableObject {
     );
   }
 
+  // shouldInclude should return true if the given language should be included.
+  // That means that either the given language is selected or the "All" option is selected.
```

Comment:
Can you use JSDoc comments for these? That will ensure documentation shows up in IDEs:

```suggestion
  /**
   * This returns true if the given language should be included.
   *
   * That means that either the given language is selected or the "All" option is selected.
   *
   * @param language
   */
```